pylatex/utils.py

```diff
@@ -272,14 +291,120 @@ def italic(s, *, escape=True):
     '\\textit{hello}'
     >>> print(italic("hello"))
     \textit{hello}
-
     """
     if escape:
         s = escape_latex(s)
 
     return NoEscape(r'\textit{' + s + '}')
 
 
+def horizontal_skip(size):
+    r"""Add/remove the amount of horizontal space between elements.
+
+    Args
+    ----
+    size: str
+        The amount of horizontal space to add
+    """
+
+    return pylatex.base_classes.UnsafeCommand("hspace*", arguments=size)
+
+
+def vertical_skip(size):
+    r"""Add the user specified amount of vertical space to the document.
+
+    Args
+    ----
+    size: str
+        The amount and units of vertical space to create
+    """
+
+    return pylatex.base_classes.UnsafeCommand("vspace*", arguments=size)
+
+
+def text_box(s):
+    r"""Add a text box around the text.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+
+    Returns
+    -------
+    ContainerCommand
+        The container with the content specified
+    """
+
+    return pylatex.base_classes.ContainerCommand(command='fbox', data=s)
+
+
+def center(s, *, escape=True):
```

Comment:
Could you remove these functions from utils, now that they are in the basic module?